containerize/Program.cs

```diff
@@ -68,13 +69,28 @@ async Task Containerize(DirectoryInfo folder, string workingDir, string registry
     Image x = await registry.GetImageManifest(baseName, baseTag);
     x.WorkingDirectory = workingDir;
 
+    JsonSerializerOptions options = new()
+    {
+        WriteIndented = true,
+    };
+
     Console.WriteLine($"Copying from {folder.FullName} to {workingDir}");
     Layer l = Layer.FromDirectory(folder.FullName, workingDir);
 
     x.AddLayer(l);
 
     x.SetEntrypoint(entrypoint);
 
+    // File.WriteAllTextAsync("manifest.json", x.manifest.ToJsonString(options));
+    // File.WriteAllTextAsync("config.json", x.config.ToJsonString(options));
+
+    await LocalDocker.Load(x, imageName, baseName);
+
+    Console.WriteLine($"Loaded image into local Docker daemon. Use 'docker run --rm -it --name {imageName} {registryName}/{imageName}:latest' to run the application.");
```

Comment:
Containerize is effectively a unit test at this point. We should write a UT that runs this and expects success.

it may need to live in the System.Containers.Filesystem test proj for docker initialization stuff